src/components/MaTextList/MaTextList.css

```
@@ -0,0 +1,57 @@
+ul,
+ol {
+  list-style: none;
+}
+
+li {
+  position: relative;
+}
+
+li p {
+  max-width: 75ch;
+}
+
+li:not(:last-child) {
+  margin-bottom: 0.5rem;
+}
+
+ul li::before {
+  content: '';
+  position: absolute;
+}
+
+ul.bullet li::before {
+  background: url('../../assets/icons/bullet.svg');
+  width: 0.625rem;
+  height: 0.625rem;
+  top: 0.5rem;
+  left: -1.375rem;
+}
+
+ul.bullet li.small::before {
+  top: 0.375rem;
+}
+
+ul.check li::before {
+  background: url('../../assets/icons/check.svg');
+  width: 1rem;
+  height: 0.75rem;
+  top: 0.5rem;
+  left: -1.5rem;
+}
+
+ul.check li.small::before {
+  top: 0.25rem;
+}
+
+ol {
+  counter-reset: counter;
+}
+
+ol li::before {
+  counter-increment: counter;
+  content: counter(counter) '.';
+  position: absolute;
+  top: 0;
+  left: -1.25rem;
+}
```

isn't all this the same as:

```suggestion
ol {
  list-style-type: decimal;
}
```

---

If I recall correctly we chose the `::before` in order to custom the spacing (because we couldn't manage to do it with `::marker`). However, after mentioning it here, maybe it's better for instance in terms of vertical alignment. So, let's use `list-style-type: decimal;` and discuss with design about it.